extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoHandler.java

```diff
@@ -81,7 +83,15 @@
         @Override
         public void run() {
             try {
+                if (!isUpnpDeviceRegistered()) {
+                    logger.warn("WeMo UPnP device {} not yet registered", getUDN());
+                    updateStatus(ThingStatus.OFFLINE);
+                    return;
+                }
+
                 updateWemoState();
+                onSubscription();
+
```

Comment:
I don't think it is a good idea to add subscriptions on every polling run, so please remove it here